packages/db-ra-data-rest/package.json

```diff
@@ -19,6 +19,7 @@
     "src"
   ],
   "scripts": {
+    "prepublish": "npm run build",
```

Comment:
is this enough? Should it use pnpm or npm?

Reply:
should be ok